src/base/constants.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Flag used to store browser/main boot up data
+ */
+export const K_BROWSER_STORAGE = "__browser_store";
```

Comment:
Why do you use `_` here but `-` for the other variables?

Reply:
ah. `__` prefix should be used here since this get's mounted on `global` object and we want to minimize collision. but `-` should be used between `browser` and `store`. good catch, will change.